File: all/cap01/casos/EP1_1.r
# Valor Fixo (conforme pede EP1_1)
C <- 65

# Cálculo
F <- C * 9/5 + 32

# sprintf garante a formatação %.1f (um decimal)
# cat imprime sem índices [1] que o print() coloca
cat(sprintf("%d graus Celsius corresponde a %.1f graus Fahrenheit\n", C, F))

File: all/cap01/casos/EP1_1.js
function escreva(s) { try { document.write(s); } catch(e) { console.log(s); } }

  var C = 65;
  var F = C * 9/5 + 32;
  escreva(C + " graus Celsius corresponde a " + F.toFixed(1) + " graus Fahrenheit");
